tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should succeed getting Deployment object")
```

Comment:
small nit, with this wording it will be aligned with all the rest of those messages we create
"Should successfully get Deployment object"